src/metric.ts

```diff
@@ -0,0 +1,27 @@
+export class Metric {
+    private readonly SECOND = 1000;
+    private static _metric: Metric;
+
+    private fpsLabel: HTMLElement;
+    private recentDraws: number[] = [];
+
+    public static getMetric(): Metric {
+        return this._metric || (this._metric = new Metric());
+    }
+
+    private constructor() {
+        this.fpsLabel = document.getElementById("fps-status")!;
```

Comment:
I can't see any change to the HTML file which would match with this new Id. Did you forget to commit/push the change to the HTML file?